backend/server.ts

```diff
@@ -21,14 +19,14 @@ app.use('/api/upload', uploadRoutes)
 
 app.use('/uploads', express.static(path.join(__dirname, '/uploads')))
 
-if (process.env.NODE_ENV === 'production') {
+if (config.shouldServeReactApp) {
```

Comment:
What an elegant solution, kudos!

Reply:
hahahahaha I learned this elegant way, did you like it?